learn/tanssi/index.md

```diff
@@ -1,8 +1,21 @@
 ---
 title: Learn About Tanssi
 description: Learn about Tanssi protocol and how it handles infrastructure complexities, making it easy for new appchains to be deployed in the Polkadot Web3 ecosystem.
-template: subsection-index-page.html
-hide: 
- - toc
- - feedback
+icon: octicons-star-24
+template: index-page.html
 ---
+
+# Learn About Tanssi
+
+Here, you'll find an overview of Tanssi’s core capabilities that support appchain development, covering everything from infrastructure and security to essential integrations.
+
+Key Features That Set Tanssi Apart:
```

Comment:
```suggestion
Key features that set Tanssi apart:
```

This isn't really a title, so we should treat it as a normal sentence that uses sentence case